contracts/package.json

```diff
@@ -5,22 +5,22 @@
   "main": "index.js",
   "scripts": {
     "compile:solidity": "hardhat compile",
-    "test": "hardhat --network localhost test"
+    "test": "hardhat test"
   },
   "author": "",
   "license": "MIT",
   "devDependencies": {
     "@ethereum-waffle/mock-contract": "^4.0.4",
     "@nomicfoundation/hardhat-chai-matchers": "^1.0.3",
     "@nomiclabs/hardhat-ethers": "^2.0.5",
-    "@shardlabs/starknet-hardhat-plugin": "^0.8.0-alpha.2",
     "@types/chai": "^4.3.3",
     "@types/elliptic": "^6.4.14",
     "@types/mocha": "^9.1.1",
     "chai": "^4.3.6",
     "ethers": "^5.6.8",
     "hardhat": "^2.16.1",
-    "solidity-coverage": "^0.8.2"
+    "solidity-coverage": "^0.8.2",
+    "starknet": "6.4.0"
```

Comment:
Already a dependency in the root package.json

Reply:
Nice catch - thank you! Should be fixed now 🚀